src/Tests/Seam/When_dispatching_messages_in_receive_context.cs

```diff
@@ -58,16 +51,29 @@ public void SetUp()
             endpointOrientedTopology.Initialize(settings);
 
             // create the topologySectionManager
-            var topologyCreator = (ICreateTopologyInternal) container.Resolve(typeof(TopologyCreator));
-            sectionManager = container.Resolve<ITopologySectionManagerInternal>();
-            await topologyCreator.Create(sectionManager.DetermineResourcesToCreate(new QueueBindings()));
-            container.RegisterSingleton<TopologyOperator>();
+            var namespaceManagerCreator = new NamespaceManagerCreator(settings);
+
+            var namespaceManagerLifeCycleManagerInternal = new NamespaceManagerLifeCycleManagerInternal(namespaceManagerCreator);
```

Comment:
Probably nitpicking here, but still 
`namespaceManagerLifeCycleManagerInternal` is `Internal` really needed?